Classes/KIFUITestActor.m

```diff
@@ -204,6 +204,10 @@ - (void)tapAccessibilityElement:(UIAccessibilityElement *)element inView:(UIView
         
         // This is mostly redundant of the test in _accessibilityElementWithLabel:
         KIFTestWaitCondition(!isnan(tappablePointInElement.x), error, @"View is not tappable");
+        
+        if ([NSStringFromClass([view class]) isEqualToString:@"_UIAlertControllerActionView"]) {
+            [view longPressAtPoint:tappablePointInElement duration:0.1];
+        }
```

Comment:
Checking the view class makes this code very fragile and prompt to fail if Apple changes the internal of UIAlertController
However we are already using this strategy in some other places. Ex: https://github.com/kif-framework/KIF/blob/master/Additions/UIView-KIFAdditions.m#L405

@phatmann and others let me know what you think. Thx